givitsite/coordinate/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from . import views
+
+
+urlpatterns = [
```

Comment:
Use upper case for consts "URL_PATTERNS"

Reply:
That variable is taken from the django documentation - https://docs.djangoproject.com/en/3.1/topics/http/urls/